### PR TITLE
bugfix/ZCS-1129 TestStoreManager

### PR DESCRIPTION
**Pull request by Vincedo84**
zimbra@mail130:/root$ zmsoap -z
RunUnitTestsRequest/test=com.zimbra.qa.unittest.TestStoreManager
<RunUnitTestsResponse numFailed="0" numSkipped="0" numExecuted="1"
xmlns="urn:zimbraAdmin">
<results>
<completed name="testStore"
class="com.zimbra.qa.unittest.TestStoreManager" execSeconds="9.74"/>
</results>
</RunUnitTestsResponse>

### CODE REVIEW

File: store/src/java/com/zimbra/qa/unittest/TestStoreManager.java [hunk 1]
```diff
@@ -47,9 +57,30 @@ public static ParsedMessage getMessage() throws Exception {
         mm.setText("nothing to see here" + RandomStringUtils.random(1024));
         return new ParsedMessage(mm, false);
     }
+	
+	 @Before
+    public void setUp()
+    throws Exception {
+		String prefix = NAME_PREFIX + "-" + testInfo.getMethodName() + "-";
```
Comment (tmclane) [inline store/src/java/com/zimbra/qa/unittest/TestStoreManager.java]: Please correct the whitespace characters to not use tabs.

File: store/src/java/com/zimbra/qa/unittest/TestStoreManager.java [hunk 2]
```diff
@@ -47,9 +57,30 @@ public static ParsedMessage getMessage() throws Exception {
...
 
+	private void cleanUp() 
+	throws Exception{
+        TestUtil.deleteAccountIfExists(USER_NAME);
+    }
+	
     @Test
     public void testStore() throws Exception {
+		
+		TestUtil.createAccount(USER_NAME);
```
Comment (tmclane) [inline store/src/java/com/zimbra/qa/unittest/TestStoreManager.java]: same here

File: store/src/java/com/zimbra/qa/unittest/TestStoreManager.java (hunk 1 above)
Comment (Vincedo84) [inline store/src/java/com/zimbra/qa/unittest/TestStoreManager.java]: Thanks Travis, I will update the script

**Comment by fatkudu** [comment]
Thanks - merged the changes.

**Pull request closed without merging**